src/components/App.tsx

```diff
@@ -900,6 +902,20 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         gridSize: this.props.gridModeEnabled ? GRID_SIZE : null,
       });
     }
+
+    if (
+      prevProps.exportName !== this.props.exportName &&
+      this.props.exportName
+    ) {
+      this.setState({
+        name: this.props.exportName,
+      });
+    }
+
+    if (this.props.onExportNameChange && prevState.name !== this.state.name) {
+      this.props.onExportNameChange(this.state.name);
+    }
```

Comment:
Not sure if this is the right way to trigger the callback

Reply:
instead of triggering here, this should be triggered from the input handler of this field instead.